### PR TITLE
Feature idva5 630/unincorporated] update design for 'what is the business name'

### PR DESCRIPTION
**Pull request by hemerem-ch**
-update h1, hint and added comment to screen

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by abhiahirv1 — DISMISSED** [review]

File: src/main/controllers/features/unincorporated/whatIsTheBusinessNameController.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     }
     res.render(config.UNINCORPORATED_WHAT_IS_THE_BUSINESS_NAME, {
         previousPage: addLangToUrl(previousPage, lang),
-        title: "What is the business name?",
+        title: "What is the name of th business?",
```
Comment (ttingle-ch) [inline src/main/controllers/features/unincorporated/whatIsTheBusinessNameController.ts:26]: Spelling mistake needs fixing

File: src/main/controllers/features/unincorporated/whatIsTheBusinessNameController.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
             const pageProperties = getPageProperties(formatValidationError(errorList.array(), lang));
             res.status(400).render(config.UNINCORPORATED_WHAT_IS_THE_BUSINESS_NAME, {
                 previousPage: addLangToUrl(previousPage, lang),
-                title: "What is the business name?",
+                title: "What is the name of th business?",
```
Comment (ttingle-ch) [inline src/main/controllers/features/unincorporated/whatIsTheBusinessNameController.ts:50]: Same as above

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]

**Pull request merged**